modules/metering-reports.adoc

```diff
@@ -0,0 +1,332 @@
+node-memory-capacity-raw                     23m
+node-memory-utilization                      23m
+persistentvolumeclaim-capacity               23m
+persistentvolumeclaim-capacity-raw           23m
+persistentvolumeclaim-phase-raw              23m
+persistentvolumeclaim-request                23m
+persistentvolumeclaim-request-raw            23m
+persistentvolumeclaim-usage                  23m
+persistentvolumeclaim-usage-raw              23m
+persistentvolumeclaim-usage-with-phase-raw   23m
+pod-cpu-request                              23m
+pod-cpu-request-raw                          23m
+pod-cpu-usage                                23m
+pod-cpu-usage-raw                            23m
+pod-memory-request                           23m
+pod-memory-request-raw                       23m
+pod-memory-usage                             23m
+pod-memory-usage-raw                         23m
+----
+
+ReportQueries with the `-raw` suffix are used by other ReportQueries to build more complex queries, and should not be used directly for reports.
+
+`namespace-` prefixed queries aggregate Pod CPU/memory requests by namespace, providing a list of namespaces and their overall usage based on resource requests.
+
+`pod-` prefixed queries are similar to 'namespace-' prefixed, but aggregate information by Pod, rather than namespace. These queries include the Pod's namespace and node.
+
+`node-` prefixed queries return information about each node's total available resources.
+
+`aws-` prefixed queries are specific to AWS. Queries suffixed with `-aws` return the same data as queries of the same name without the suffix, and correlate usage with the EC2 billing data.
+
+The `aws-ec2-billing-data` report is used by other queries, and should not be used as a standalone report. The `aws-ec2-cluster-cost` report provides a total cost based on the nodes included in the cluster, and the sum of their costs for the time period being reported on.
+
+For a complete list of fields each report query produces, use `kubectl` to get the object as YAML, and check the `columns` field:
+
+For example, run:
+
+----
+kubectl -n $METERING_NAMESPACE get reportqueries namespace-memory-request -o yaml
+----
+
+You should get something like this:
+
+----
+apiVersion: metering.openshift.io/v1
+kind: ReportQuery
+metadata:
+  name: namespace-memory-request
+  labels:
+    operator-metering: "true"
+spec:
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: namespace
+    type: varchar
+    unit: kubernetes_namespace
+  - name: pod_request_memory_byte_seconds
+    type: double
+    unit: byte_seconds
+----
+
+== schedule
+
+The `spec.schedule` configuration block defines when the report runs.
+The main fields in the `schedule` section are `period`, and then depending on the value of `period`, the fields `hourly`, `daily`, `weekly` and `monthly` allow you to fine-tune when the report runs.
```

Comment:
, and `monthly`
s/fine-tune/explicitly set

Reply:
I added the comma.

Reply:
@lamek, @ahardin-rh also suggest to change `fine-tune` to `explicitly set`